Expressions on UniqueConstraint aren't supported

# Bug report

<!--
Hi, thanks for submitting a bug. We appreciate that.

But, we will need some information about what's wrong to help you.
-->

## What's wrong

Django 4.0 added support for a [positional `expressions` argument](https://docs.djangoproject.com/en/4.0/ref/models/constraints/#expressions), but it doesn't look like it's supported here yet.

Example:

```
class User:
    ...
    class Meta:
        constraints = [
            models.UniqueConstraint(Upper("email"), name="email_key"),
        ]
```

Output:

```
mypy .
xxx/models.py:74: error: Too many positional arguments for "UniqueConstraint"
xxx/models.py:74: error: Argument 1 to "UniqueConstraint" has incompatible type "Upper"; expected "Sequence[str]"
xxx/migrations/0001_initial.py:111: error: Too many positional arguments for "UniqueConstraint"
xxx/migrations/0001_initial.py:112: error: Argument 1 to "UniqueConstraint" has incompatible type "Upper"; expected "Sequence[str]"
```


<!--
Describe what is not working.

Please, attach a traceback.
We would also appreciate a failing test case.
That is EXTREMELY helpful!
-->




## How is that should be

Shouldn't output errors.

## System information

- OS: Mac OS
- `python` version: 3.10
- `django` version: 4.0
- `mypy` version: 0.930
- `django-stubs` version: 1.9.0
- `django-stubs-ext` version: 0.3.1

--- a/django-stubs/db/models/constraints.pyi
+++ b/django-stubs/db/models/constraints.pyi
@@ -1,8 +1,9 @@
 import enum
-from typing import Any, Optional, Sequence, Tuple, Type, TypeVar
+from typing import Any, Optional, Sequence, Tuple, Type, TypeVar, Union
 
 from django.db.backends.base.schema import BaseDatabaseSchemaEditor
 from django.db.models.base import Model
+from django.db.models.expressions import Combinable
 from django.db.models.query_utils import Q
 
 _T = TypeVar("_T", bound="BaseConstraint")
@@ -27,9 +28,15 @@ class CheckConstraint(BaseConstraint):
     def __init__(self, *, check: Q, name: str) -> None: ...
 
 class UniqueConstraint(BaseConstraint):
+    expressions: Tuple[Combinable]
     fields: Tuple[str]
     condition: Optional[Q]
     deferrable: Optional[Deferrable]
     def __init__(
-        self, *, fields: Sequence[str], name: str, condition: Optional[Q] = ..., deferrable: Optional[Deferrable] = ...
+        self,
+        *expressions: Union[str, Combinable],
+        fields: Optional[Sequence[str]] = ...,
+        name: str,
+        condition: Optional[Q] = ...,
+        deferrable: Optional[Deferrable] = ...,
     ) -> None: ...

--- a/django-stubs/db/models/constraints.pyi
+++ b/django-stubs/db/models/constraints.pyi
@@ -29,7 +29,7 @@ class CheckConstraint(BaseConstraint):
 
 class UniqueConstraint(BaseConstraint):
     expressions: Tuple[Combinable, ...]
-    fields: Tuple[str]
+    fields: Tuple[str, ...]
     condition: Optional[Q]
     deferrable: Optional[Deferrable]
     def __init__(

--- a/django-stubs/db/models/constraints.pyi
+++ b/django-stubs/db/models/constraints.pyi
@@ -28,7 +28,7 @@ class CheckConstraint(BaseConstraint):
     def __init__(self, *, check: Q, name: str) -> None: ...
 
 class UniqueConstraint(BaseConstraint):
-    expressions: Tuple[Combinable]
+    expressions: Tuple[Combinable, ...]
     fields: Tuple[str]
     condition: Optional[Q]
     deferrable: Optional[Deferrable]

